src/entrypoints/entrypoints.ts

```diff
@@ -49,17 +48,14 @@ class NetworkEntrypoint {
     }
 
     async signTransaction(transaction: Transaction, account: IAccount): Promise<void> {
-        const txComputer = new TransactionComputer();
-        transaction.signature = await account.sign(txComputer.computeBytesForSigning(transaction));
+        transaction.signature = await account.signTransaction(transaction);
     }
 
-    verifyTransactionSignature(transaction: Transaction): boolean {
-        const verifier = UserVerifier.fromAddress(transaction.sender);
-        const txComputer = new TransactionComputer();
-        return verifier.verify(txComputer.computeBytesForVerifying(transaction), transaction.signature);
+    async verifyTransactionSignature(transaction: Transaction, account: IAccount): Promise<boolean> {
+        return await account.verifyTransaction(transaction, transaction.signature);
     }
 
-    verifyMessageSignature(message: Message): boolean {
+    async verifyMessageSignature(message: Message, account: IAccount): Promise<boolean> {
```

Comment:
In account, we have `verifyTransaction` and `verifyMessage`. Here, we also have the suffix `signature`. Keep it or drop it? (either way should be fine, but we must also reflect this in the specs).

Reply:
I kind of like having `signature` in the name, but if we decide to drop it, it's fine